src/ordering.f90

```diff
@@ -69,30 +69,30 @@ pure subroutine get_index_reordering(out_i, out_j, out_k, in_i, in_j, in_k, reor
       integer :: dir_in, dir_out
 
       select case (reorder_dir)
-         case (RDR_X2Y)
-            dir_in = DIR_X
-            dir_out = DIR_Y
-         case (RDR_X2Z)
-            dir_in = DIR_X
-            dir_out = DIR_Z
-         case (RDR_Y2X)
-            dir_in = DIR_Y
-            dir_out = DIR_X
-         case (RDR_Y2Z)
-            dir_in = DIR_Y
-            dir_out = DIR_Z
-         case (RDR_Z2X)
-            dir_in = DIR_Z
-            dir_out = DIR_X
-         case (RDR_Z2Y)
-            dir_in = DIR_Z
-            dir_out = DIR_Y
+       case (RDR_X2Y)
```

Comment:
Another interesting point. `case` is shifted only one space, not two, and also the lines inside the `case` are shifted by one.